force_nevergrad/mco/ng_mco_factory.py

```diff
@@ -0,0 +1,48 @@
+from force_bdss.api import (
+    BaseMCOFactory,
+    FixedMCOParameterFactory,
+    ListedMCOParameterFactory,
+    RangedMCOParameterFactory,
+    CategoricalMCOParameterFactory,
+    BaseMCOCommunicator
+)
+
+from .ng_mco_model import NevergradMCOModel
+from .ng_mco import NevergradMCO
+
+
+class MCOCommunicator(BaseMCOCommunicator):
+    def send_to_mco(self, model, kpi_results):
+        pass
+
+    def receive_from_mco(self, model):
+        pass
+
+
+class NevergradMCOFactory(BaseMCOFactory):
+    def get_identifier(self):
+        return "nevergrad_mco"
+
+    def get_name(self):
+        return "Gradient Free Multi Criteria optimizer"
+
+    #: Returns the model class
+    def get_model_class(self):
+        return NevergradMCOModel
+
+    #: Returns the optimizer class
+    def get_optimizer_class(self):
+        return NevergradMCO
+
+    #: Returns the communicator class
+    def get_communicator_class(self):
+        return MCOCommunicator
```

Comment:
We don't expect to use the `MCOCommunicator` class, so if we return `BaseMCOCommunicator` here then I believe this will only cause an issue if an instance is created using the `create_communicator` method. In which case the developer will be alerted to the need to implement a custom subclass with both `send_to_mco` and `receive_from_mco` methods.

Reply:
Thank you, great idea. I thought it would fail and added the dummy one.